loadtest/user/userentity/custom_profile_attributes.go

```diff
@@ -0,0 +1,49 @@
+package userentity
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+// CreateCPAField creates a new custom profile attribute field.
+func (ue *UserEntity) CreateCPAField(field *model.PropertyField) (*model.PropertyField, error) {
+	new_field, _, err := ue.client.CreateCPAField(context.Background(), field)
+	if err != nil {
+		return nil, err
+	}
+	return new_field, nil
+}
+
+// GetCPAFields retrieves all the cpa field values available.
```

Comment:
Typo? Still not sure about terminology :joy: 
```suggestion
// GetCPAFields retrieves all the cpa fields available.
```